compiler/rustc_query_system/src/query/plumbing.rs

```diff
@@ -381,7 +381,7 @@ where
     })
 }
 
-#[inline(always)]
+#[inline]
```

Comment:
It might be worth (temporarily) reverting this commit. If possible, I would like to have some idea of where the performance regressions perf.rlo is showing are coming from. If we can narrow it down, then we can evaluate more clearly if the lost performance is worth the upsides.